packages/functional-tests/tests/subscription-tests/coupons-ui-tests/resubscription.spec.ts

```diff
@@ -75,6 +90,11 @@ test.describe('resubscription test', () => {
     expect(await subscriptionManagement.getResubscriptionPrice()).toEqual(
       total
     );
+
+    //Verify no coupon details are visible
+    expect(await subscriptionManagement.subscriptionDetails()).not.toContain(
+      'Promo'
+    );
```

Comment:
nit: I think we use this pattern in 4&times; places. Not sure if we want to refactor into something like:
```js
expect(await subscriptionManagement.subscriptionIsPromo()).toBeFalsy();
```
That way if the text messaging changes, you'd only need to update 1 helper instead of _n_ tests.

Reply:
I am not sure how I can refactor this considering I am checking in 4 different places for the "Promo" text. But yeah something for me to think and update in my later PRs. Thanks Peter for the suggestion! :)